students/dae-hwan/user/views.py

```diff
@@ -0,0 +1,50 @@
+import json 
+
+from django.http  import JsonResponse
+from django.views import View
+
+from user.models  import User
+
+class SignUpView(View):
+    def post(self, request):
+        data                = json.loads(request.body)
+        email               = data['email']
+        password            = data['password']
+        nick_name           = data['nick_name']
+        phone_number        = data['phone_number']
+        email_validation    = '^[a-z0-9,_-.]+@[a-z0-9,_-]+\.[a-z0-9,_-.]+$'
+        password_validation = '.{8,}'
+
+        # 정보가 잘 입력되었을 때
+        try: 
+            # 이메일에 관한 에러
+            if '@' not in email or '.' not in email:
+                return JsonResponse({'message': 'email_validation'}, status = 400)
+            
+            # 비밀번호에 관한 에러
+            min_password = 8
+            if len(password) < min_password:
+                return JsonResponse({'message': 'password validation'}, status = 400)
+        
+            # 닉네임에 관한 에러
+            if User.objects.filter(nick_name = nick_name).exists():
+                return JsonResponse({'message': 'nick_name exist'}, status = 400)
+
+            # 전화번호에 관한 에러
+            if User.objects.filter(phone_number = phone_number).exists():
+                return JsonResponse({'message': 'phone_number exist'}, status = 400)
+
+            # email이 중복되지 않을시 데이터 생성
+            if User.objects.filter(email = email).exists():
+                return JsonResponse({'message': 'email exist'}, status = 400)
+            User.objects.create(
```

Comment:
```suggestion
                return JsonResponse({'message': 'email exist'}, status = 400)
                
            User.objects.create(
```
공백을 한 칸 주는 것이 가독성에 더 좋을 것 같아요